marimo/dagger/src/main/env.py

```
@@ -0,0 +1,76 @@
+            .with_mounted_cache(
+                "/root/.local/share/pnpm", dag.cache_volume("pnpm")
+            )
+            # package deps
+            .with_exec(["apt", "update"])
+            .with_exec(["apt", "install", "-y", "curl"])
+            # install node 20+
+            .with_exec(
+                [
+                    "sh",
+                    "-c",
+                    "curl -fsSL https://deb.nodesource.com/setup_20.x | bash -",  # noqa: E501
```

Ruff (0.14.0)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)